js/data/bucket.js

```diff
@@ -58,8 +58,6 @@ class Bucket {
                 options.featureIndex.insert(feature, this.index);
```

Comment:
Feature querying for symbols does not use `FeatureIndex`. Instead it uses the `CollisionTile` generated during placement. If you're seeing issues with symbol querying, please file a separate issue.